src/Csv.php

```diff
@@ -889,10 +889,30 @@ public function load_data($input = null) {
 
         if (is_null($input)) {
             $file = $this->file;
-        } elseif (file_exists($input)) {
-            $file = $input;
         } else {
-            $data = $input;
+            $inputLengthCondition = mb_strlen($input) > PHP_MAXPATHLEN;
```

Comment:
1. this is a rather odd name for the variable, IMHO. Also, the code style of this particular library doesn't use camelCase, so let's not mix styles unnecessarily.
2. I believe PHP_MAXPATHLEN is in bytes, not in characters.